mass/adapters/inbound/event_sub.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Event subscriber details for searchable resource events"""
+import ghga_event_schemas.pydantic_ as event_schemas
+from ghga_event_schemas.validation import get_validated_payload
+from hexkit.custom_types import Ascii, JsonObject
+from hexkit.protocols.eventsub import EventSubscriberProtocol
+from pydantic import BaseSettings, Field
+
+from mass.core.models import Resource
+from mass.ports.inbound.query_handler import QueryHandlerPort
+
+
+class EventSubTranslatorConfig(BaseSettings):
+    """Config for the event subscriber"""
+
+    searchable_resource_events_topic: str = Field(
```

Comment:
```suggestion
    resource_change_event_topic: str = Field(
```
Would keep the name similar to what is used for type to make it easier to see what belongs together directly in the config.

Reply:
Changed in [baf1354](https://github.com/ghga-de/mass/pull/12/commits/baf135406fb6ee1cec93a58685ac321d490b7a61)